Makefile

```diff
@@ -126,7 +122,8 @@ fmt: ## Run go fmt against code.
 vet: ## Run go vet against code.
 	go vet ./...
 
-container-unit-test: ## Build docker image with the manager and execute unit tests.
+container-unit-test: VERSION ?= $(shell cat .version)
```

Comment:
What do lines 125 and 126 do here?

Reply:
Line 126 makes sure the ".version" prerequisite runs before the "container-unit-test" recipe runs.
Line 127 sets the VERSION variable after the .version file is created, and before the "container-unit-test" recipe runs.

Reply:
https://www.gnu.org/software/make/manual/html_node/Target_002dspecific.html